packages-content-model/roosterjs-content-model-editor/lib/modelApi/common/mergeModel.ts

```diff
@@ -131,6 +133,9 @@ function mergeParagraph(
 
     if (newPara.decorator) {
         newParagraph.decorator = { ...newPara.decorator };
+        if (HeadingTags.indexOf(newParagraph.decorator.tagName) > -1) {
+            delete newParagraph.segmentFormat;
```

Comment:
do we really need to delet ethe whole segmentFormat? Or we just want to delete the conflict ones?

Reply:
Let me verify this. I think in theory it should be okay to remove all, at least with the testing I did. But if we can keep some properties would be better.

Reply:
Just tested and I think it should be okay to just remove the conflict formats. Added that to the latest commits